src/melee.cpp

```diff
@@ -1229,14 +1229,22 @@ void player::perform_technique( const ma_technique &technique, Creature &t, dama
     }
 
     if( technique.knockback_dist > 0 ) {
+        const tripoint prev_pos = t.pos(); // track target startpoint for knockback_follow
         const int kb_offset_x = rng( -technique.knockback_spread,
                                      technique.knockback_spread );
         const int kb_offset_y = rng( -technique.knockback_spread,
                                      technique.knockback_spread );
+                                     
         tripoint kb_point( posx() + kb_offset_x, posy() + kb_offset_y, posz() );
         for( int dist = rng( 1, technique.knockback_dist ); dist > 0; dist-- ) {
             t.knock_back_from( kb_point );
         }
+        // This technique makes the player follow into the tile the target was knocked from
+        if (technique.knockback_follow > 0 ) {
```

Comment:
Until you get astyle/git set up on your debian machine as you said...
`if ( technique.knockback_follow > 0 ) {`

Reply:
Thanks.